tests/ReflectionClassTest.php

```diff
@@ -47,21 +48,21 @@ public function testGetModifiers($fileName)
     /**
      * Performs method-by-method comparison with original reflection
      *
-     * @dataProvider caseProvider
+     * @dataProvider methodCaseProvider
      *
-     * @param ReflectionClass   $parsedClass Parsed class
-     * @param \ReflectionMethod $refMethod Method to analyze
-     * @param string                  $getterName Name of the reflection method to test
+     * @param \ReflectionClass $refClass Original reflection class
+     * @param ReflectionClass $parsedClass Parsed class
+     * @param string $getterName Name of the reflection method to test
      */
     public function testReflectionMethodParity(
+        \ReflectionClass $refClass,
         ReflectionClass $parsedClass,
         $getterName
     ) {
         $className = $parsedClass->getName();
-        $refClass  = new \ReflectionClass($className);
 
-        $expectedValue = $refClass->$getterName();
-        $actualValue   = $parsedClass->$getterName();
+        $expectedValue = $refClass->{$getterName}();
```

Comment:
Is this and next change really needed? 

Should work fine on both PHP5 and PHP7 already.